crypto/cmake/crypto-config.cmake

```diff
@@ -1,4 +1,4 @@
-if(WIN32 OR UNIX OR APPLE)
+if(WIN32 OR UNIX)
```

Comment:
how is this related to enabling openbsd?

Reply:
It's related to an earlier comment from Justin here `APPLE` and `UNIX` were used in a cmake check, but UNIX covers both, so this was the only other spot where we could simplify. Not BSD specific.